defender/forta_large_delegations/serverless.yml

```diff
@@ -32,43 +32,56 @@ defender:
 
 functions:
   autotask-1:
-    name: ${self:custom.name} Serverless Autotask
-    path: autotask-1
+    name: Forta Large Delegations Filter
     trigger:
-      type: schedule
-      frequency: 525600000
+      type: webhook
```

Comment:
we should avoid creating webhooks for autotasks that are triggered from sentinels whether as autotask conditions or notification autotasks.  I prefer we use the latest version of the '@openzeppelin/defender-serverless' plugin which should include the new sentinel trigger type.  This depends on assumption that autotask handler arguments change by how it's called, condition vs notification, not how the trigger is configured.

Reply:
Even after updating the package I'm still getting 
```
Configuration error:
     at 'functions.autotask-1.trigger.type': must be equal to one of the allowed values [schedule, webhook]`

```

Reply:
addressed with bug report, possible PR to follow